packages/filesystem/src/common/filesystem-watcher.ts

```diff
@@ -29,21 +28,13 @@ export class FileSystemWatcher implements Disposable {
 
     constructor(
         @inject(FileSystemWatcherServer) protected readonly server: FileSystemWatcherServer,
-        @inject(FileSystemPreferences) protected readonly preferences: FileSystemPreferences
     ) {
         this.toDispose.push(this.onFileChangedEmitter);
 
         this.toDispose.push(server);
         server.setClient({
             onDidFilesChanged: e => this.onDidFilesChanged(e)
         });
-
```

Comment:
Is `FileSystemWatcher` used anywhere in the backend? if not then just move fs preferences and the service to the browser folder

Reply:
Good idea, I think it's only used in file-resource which is in src/common, but this one could be moved to frontend as well wdyt?